libraries/message/src/cli_to_coordinator.rs

```diff
@@ -64,12 +64,14 @@ pub enum ControlRequest {
         grace_duration: Option<Duration>,
     },
     Logs {
-        uuid: Option<Uuid>,
-        name: Option<String>,
-        node: String,
+        uuid: Uuid,
+        node: NodeId,
     },
     Destroy,
     List,
+    Info {
+        dataflow_uuid: Uuid,
+    },
```

Comment:
I would count this change as backwards-compatible since it's a new command. The behavior would be the following:

- if only CLI is updated: coodinator sends a `ControlRequestReply::Error` back, which will contain the deserialization error. Not nice, but ok.
- if only coordinator is updated: fine, as the CLI never sends the new message
- if both CLI and coordinator are updated: fine